### PR TITLE
Code cleanup

### LINKED ISSUE #137: Use `Ember.$` instead of `$`

### PR DESCRIPTION
**Pull request by ConnorCimowsky**
* Remove unnecessary gitkeep files
* Remove outdated fixture data
* Convert all indentation to 4 spaces
* Use Ember.$ instead of $

:sparkles: Resolves #137 :sparkles:

### CODE REVIEW

File: src-frontend/app/components/page-list.js [hunk 1]
```diff
@@ -13,8 +12,8 @@ export default Ember.Component.extend({
             update: function(event, ui) {
                 var indices = {};
 
-                $(this).find('a.open-page').each(function(index) {
-                    indices[$(this).data('id')] = index;
+                Ember.$(this).find('a.open-page').each(function(index) {
```
Comment (Trinovantes) [inline src-frontend/app/components/page-list.js]: Ember has its own internal jQuery object?

**Comment by Trinovantes** [comment]
+1

File: src-frontend/app/components/page-list.js (hunk 1 above)
Comment (ConnorCimowsky) [inline src-frontend/app/components/page-list.js]: Yup, and for performance reasons you're supposed to use it instead of the global one.

Comment (ConnorCimowsky) [inline src-frontend/app/components/page-list.js]: Hm it looks like it's just an alias. But it eliminates the need for an import, so still worth using IMO

Comment (Trinovantes) [inline src-frontend/app/components/page-list.js]: sounds good

**Comment by Ommy** [comment]
LGTM

**Pull request merged**